### DISCUSSION TITLE
Does it support Background Delivery in HealthKit ?

### DISCUSSION THREAD

**Discussion by pandey21197** [doville/rn-fitness-tracker#110] [General]
Can we fetch the data when app is closed for both android and ios. Something similar to Health kit background delivery ?

**Comment by alarm109** [comment]
Hi @pandey21197,
The module was tested in the active state of the application. Therefore, I can't really tell you what happens when the application is terminated.
Also could you specify your question, do you want to fetch data when the app is terminated? Or do you wish to start fetching data in the active state and then finish in background?
